<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="buttonControlBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>321</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Sterowanie przyciskami</string>
    </property>
    <widget class="QPushButton" name="driveRightButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Prawo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="driveForawrdButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Do przodu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="driveLeftButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Lewo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="driveBackwardButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Do tylu</string>
     </property>
    </widget>
    <widget class="QSlider" name="speedSlider">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>16</width>
       <height>160</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="turningSlider">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="speedLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>200</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Predkosc</string>
     </property>
    </widget>
    <widget class="QLabel" name="turningLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Skrecanie</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Tryb</string>
    </property>
    <widget class="QRadioButton" name="buttonsMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Przyciski</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="slidersMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Slidery</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="keyboardMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Klawiatura</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>234</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="logsMonitor">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>329</y>
      <width>771</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="logsMonitorLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Monitor logow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearMonitorButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>300</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyczysc</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>211</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Sterowanie sliderami</string>
    </property>
    <widget class="QSlider" name="directionSlider">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>31</width>
       <height>141</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="sliderDirectionLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Przod/tyl</string>
     </property>
    </widget>
    <widget class="QSlider" name="turningSlider_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="turningSliderLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Skrecanie</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="leftSpeedIndicator">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>64</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="rightSpeedIndicator">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>64</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="frontRadarLCD">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="rearRadarLCD">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="frontRadarLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Przod</string>
    </property>
   </widget>
   <widget class="QLabel" name="rearRadarLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tyl</string>
    </property>
   </widget>
   <widget class="QLabel" name="cmLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>cm</string>
    </property>
   </widget>
   <widget class="QLabel" name="cmLabel_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>cm</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
